hydra_provider/proto/data_feed.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package data_feed;
+
+service SerialDataFeed {
+	rpc start(Empty) returns(Empty);
+	rpc stop(Empty) returns(Empty);
+	rpc list_available_ports(Empty) returns(ListAvailablePortsResponse);
+	rpc configure(SerialDataFeedConfig) returns(Empty);
+	rpc get_status(Empty) returns(SerialDataFeedStatus);
+}
+
+message ListAvailablePortsResponse {
+	repeated string ports = 1;
+}
+message SerialDataFeedConfig {
+	string port = 1;
+	int32 baud_rate = 2;
+}
+message SerialDataFeedStatus {
+	bool is_running = 1;
+	SerialDataFeedConfig config = 2;
+}
+
+service RandomDataFeed {
+	rpc start(Empty) returns(Empty);
+	rpc stop(Empty) returns(Empty);
+}
+
+// SHOULD DO: implement
```

Comment:
Create an issue for this.

Reply:
Done!

Reply:
https://github.com/uorocketry/rgs/issues/121